x/emissions/genesis.go

```diff
@@ -19,7 +23,7 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 // ExportGenesis returns the emissions module's exported genesis.
 func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	var genesis types.GenesisState
-	genesis.Params = k.GetParamsIfExists(ctx)
+	genesis.Params, _ = k.GetParams(ctx)
```

Comment:
I would suggest to manually set empty params if not found
We should assume to not use the return params value if found is false